var/spack/repos/builtin/packages/butterflypack/package.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Butterflypack(CMakePackage):
+    """ButterflyPACK is a mathematical software for rapidly solving
+    large-scale dense linear systems that exhibit off-diagonal rank-deficiency.
+    These systems arise frequently from boundary element methods, or
+    factorization phases in finite-difference/finite-element methods.
+    ButterflyPACK relies on low-rank or butterfly formats under Hierarchical
+    matrix, HODLR or other hierarchically nested frameworks to compress,
+    factor and solve the linear system in quasi-linear time. The
+    computationally most intensive phase, factorization, is accelerated via
+    randomized linear algebras. The butterfly format, originally inspired by
+    the butterfly data flow in fast Fourier Transform, is a linear algebra tool
+    well-suited for compressing matrices arising from high-frequency wave
+    equations or highly oscillatory integral operators."""
+
+    homepage = "https://github.com/liuyangzhuan/ButterflyPACK"
+    # url      = ""
```

Comment:
Could you remove this line?

Reply:
I have committed the change. Could you try it again?